test/test_prototype_transforms_functional.py

```diff
@@ -11,19 +11,31 @@
 make_tensor = functools.partial(torch.testing.make_tensor, device="cpu")
 
 
-def make_image(size=None, *, color_space, extra_dims=(), dtype=torch.float32):
+def make_image(size=None, *, color_space, extra_dims=(), dtype=torch.float32, constant_alpha=True):
     size = size or torch.randint(16, 33, (2,)).tolist()
 
-    num_channels = {
-        features.ColorSpace.GRAYSCALE: 1,
-        features.ColorSpace.RGB: 3,
-    }[color_space]
+    try:
+        num_channels = {
+            features.ColorSpace.GRAYSCALE: 1,
+            features.ColorSpace.GRAYSCALE_ALPHA: 1,
+            features.ColorSpace.RGB: 3,
+            features.ColorSpace.RGBA: 3,
+        }[color_space]
+    except KeyError as error:
+        raise pytest.UsageError() from error
 
     shape = (*extra_dims, num_channels, *size)
-    if dtype.is_floating_point:
-        data = torch.rand(shape, dtype=dtype)
-    else:
-        data = torch.randint(0, torch.iinfo(dtype).max, shape, dtype=dtype)
+    max_value = 1 if dtype.is_floating_point else torch.iinfo(dtype).max
```

Comment:
I assume this is where Victor's work at #5566 will come in once merged?

Reply:
Here and as well as in the kernel implementation.